src/main/java/com/datastax/cdm/job/DiffJobSession.java

```diff
@@ -255,7 +288,12 @@ private String isDifferent(EnhancedPK pk, Row originRow, Row targetRow) {
                     } else {
                         throw new RuntimeException("Target column \"" + targetColumnNames.get(targetIndex) + "\" at index " + targetIndex + " cannot be found on Origin, and is neither a constant column (indexes:" + constantColumnIndexes + ") nor an explode map column (keyIndex:" + explodeMapKeyIndex + ", valueIndex:" + explodeMapValueIndex + ")");
                     }
+                    String targetColumnType = targetColumnNames.get(targetIndex);
 
+                    if (targetColumnType.toUpperCase().contains("TTL")) {
```

Comment:
i guess this won't apply since value comparison will be skipped.. but in general just checking TTL in name is not great way..

Reply:
I agree @ashutosh-mimani . Will update the logic here.